src/branch.c

```diff
@@ -492,6 +492,14 @@ int git_branch_upstream_remote(git_buf *buf, git_repository *repo, const char *r
 	return error;
 }
 
+int git_branch_upstream_remote(git_buf *buf, git_repository *repo, const char *refname) {
```

Comment:
For functions, we would put the brace on a new line.

```suggestion
int git_branch_upstream_remote(git_buf *buf, git_repository *repo, const char *refname)
{
```